### PR TITLE
fix: workspace issues[]

### PR DESCRIPTION
**Pull request by LordNayan**
## Description

-> Fixed below workspace issues,
1) New tab not opening on new workspace creation in case any existing tab is present.
2) Loaded not appearing on Add new workspace card in workspace grid.

-> Made spinner loader responsive inside button component.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/lib/components/collections/collection-tab/MyCollection.svelte [hunk 1]
```diff
@@ -131,28 +134,40 @@
...
+          class="bg-backgroundColor input-outline form-control border-0 text-left w-100 ps-2 py-0 fs-5"
+          on:input={(event) => {
+            handleCollectionInput(event);
+          }}
+          on:blur={onRenameBlur}
+          on:keydown={onRenameInputKeyPress}
+          bind:this={inputElement}
+        />
+
+        <button
```
Comment (parthdalaltech) [inline src/lib/components/collections/collection-tab/MyCollection.svelte:161]: check if we can use button component

File: src/lib/components/collections/folder-tab/MyFolder.svelte [hunk 2]
```diff
@@ -123,44 +128,66 @@
...
+          on:input={(event) => {
+            handleFolderInput(event);
+          }}
+          maxlength={100}
+          on:blur={onRenameBlur}
+          on:keydown={onRenameInputKeyPress}
+          bind:this={inputElement}
+        />
+
+        <button
```
Comment (parthdalaltech) [inline src/lib/components/collections/folder-tab/MyFolder.svelte:155]: check for button component to be used everywhere

File: src/lib/components/collections/req-res-section/sub-components/sub-components-header/PageHeader.svelte [hunk 3]
```diff
@@ -250,37 +253,50 @@
...
+              disabled={componentData?.save ||
+                !hasWorkpaceLevelPermission(
+                  loggedUserRoleInWorkspace,
+                  workspaceLevelPermissions.SAVE_REQUEST,
+                )}
+              style="width:140px;"
+              class="save-request-btn btn btn-primary d-flex align-items-center py-1.6 justify-content-center rounded border-0"
+              on:click={() => {
+                if (
+                  componentData?.path.collectionId &&
```
Comment (parthdalaltech) [inline src/lib/components/collections/req-res-section/sub-components/sub-components-header/PageHeader.svelte:273]: add conditional operator ? fo worspaceid and collection id both

File: src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte [hunk 4]
```diff
@@ -266,9 +270,7 @@
     border: 1px solid var(--workspace-hover-color);
```
Comment (parthdalaltech) [inline src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte:270]: border-1 bootstrap class

File: src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte [hunk 5]
```diff
@@ -266,9 +270,7 @@
...
   }
 
-  .add-new-workspace {
-    border-radius: 8px;
-    background: transparent;
+  :global(.add-new-workspace) {
     border: 2px dashed var(--gradiant-2, #6147ff);
```
Comment (parthdalaltech) [inline src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte:274]: border-2 bootstrap class

File: src/lib/components/dropdown/RequestDropdown.svelte [hunk 6]
```diff
@@ -64,39 +68,47 @@
...
+        workspaceLevelPermissions.ADD_COLLECTIONS,
+      )}
+      id="dropdown-btn-color"
+      class="dropdown border-0 dropdown-btn btn p-0 d-flex align-items-center justify-content-center bg-backgroundDark {visibilty
+        ? 'drop-active'
+        : ''}"
+      style="width: 32px; height:32px;"
+      on:click={() => {
+        visibilty = !visibilty;
+        changeBtnBackground();
```
Comment (parthdalaltech) [inline src/lib/components/dropdown/RequestDropdown.svelte:90]: changebtnbackground if used at multiple places extract it out as util function

**Review by parthdalaltech — DISMISSED** [review]

**Review by punitkashyup — APPROVED** [review]

**Pull request merged**